clang/include/clang/Format/Format.h

```diff
@@ -2432,9 +2431,25 @@ struct FormatStyle {
   ///    label2:                                label2:
   ///      return 1;                              return 1;
   ///    }                                      }
+  ///
+  ///    GLI_HalfIndent:
+  ///    int f() {
+  ///      if (foo()) {
+  ///      label1:
+  ///        bar();
+  ///      }
+  ///    label2:
+  ///      return 1;
+  ///    }
   /// \endcode
   /// \version 10
-  bool IndentGotoLabels;
+  enum GotoLabelIndentation {
+    GLI_None,   // Do not indent goto labels.
+    GLI_Indent, // Indent goto labels at the same level as the surrounding code.
+    GLI_HalfIndent, // Indent goto labels at a half indent level.
```

Comment:
You need more examples (with more indentation) and/or more description what you intend to do. Do you round up or down for odd indentation levels? Do you really want that, or do you only want one level less?

``` c++
namespace NS {
  class C {
    void foo() {
      while (x) {
        if (y) {
label: //where would this be?
        }
      }
    }
  }
}
```

Reply:
doing it

Reply:
I am testing the  changes

Reply:
`3047 |   Style.IndentGotoLabels = false;` in `llvm-project/clang/unittests/Format/FormatTest.cpp`

I missed this and it's shameful correcting it